src/stateless/slider.vue

```diff
@@ -0,0 +1,597 @@
+<template>
+    <div class="slider-container" :class="Object.assign(sizeClass(), themeClass())">
```

Comment:
`:class="[theme, size] | prefix('slider---')"` and you can remove sizeClass and themeClass methods.